1-exercises/B-setting-values/exercise1.js

```diff
@@ -22,8 +22,11 @@ let capitalCities = {
     - Add a population of 9750000 to Peru's capital city.
 */
 
-// write code here
-
+capitalCities.UnitedKingdom.population = 8980000;
+capitalCities.China.population = 21500000;
+capitalCities.Peru ={};
+capitalCities.Peru.name = "Lima";
+capitalCities.Peru.population = 9750000;
 console.log(capitalCities);
 
```

Comment:
instead of adding an empty variable and then adding values to it you can add the new country and its properties in one statement
`capitalCities.peru = { name: "Lima", population: 9750000 };`